bugbot/rules/perfalert_resolved_regression.py

```diff
@@ -0,0 +1,107 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+
+from libmozdata.bugzilla import Bugzilla
+
+from bugbot.bzcleaner import BzCleaner
+
+
+class PerfAlertInactiveRegression(BzCleaner):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.extra_email_info = {}
+        self._bug_history = {}
+        self._bug_comments = {}
+
+    def description(self):
+        return "PerfAlert regressions whose resolution has changed recently"
+
+    def get_bz_params(self, date):
+        fields = [
+            "id",
+            "resolution",
+        ]
+
+        # Find all bugs that have perf-alert, and regression in their keywords. Search
+        # for bugs that have been changed in the last day. Only look for bugs after
+        # October 1st, 2024 to prevent triggering comments on older performance regressions
+        params = {
+            "include_fields": fields,
+            "f3": "creation_ts",
+            "o3": "greaterthan",
+            "v3": "2024-10-01T00:00:00Z",
+            "f1": "regressed_by",
+            "o1": "isnotempty",
+            "f2": "keywords",
+            "o2": "allwords",
+            "v2": ["regression", "perf-alert"],
+            "f4": "resolution",
+            "o4": "changedafter",
+            "v4": date,
```

Comment:
@suhaibmujahid I'd like to capture all the resolution changes made to perfalert bugs in the last day, but I'm not quite sure if this will consistently capture them or if I should use a different setting here? I'm thinking of changing this to `date - 1day` to make sure we don't miss any.

Reply:
I would use `-2d` as the value, and Bugzilla will take care of figuring out the exact date.  You can avoid processing the same bug multiple times by adding  `max_days_in_cache` to the tool config. 

Example: https://github.com/mozilla/bugbot/blob/87497dc1809ba5525bf264311f0e60ff444d0839/configs/rules.json#L425

We will run the rule daily, so it will send emails with bugs since the previous run. If you do not want to send emails on the weekends, I would increase the value to `-4d`.

Reply:
I've changed this to look at the previous day instead of using the `date` parameter.